### PR TITLE
Vision Transformers for classification and transfer learning

### PR DESCRIPTION
**Pull request by sayakpaul**
@MorganR @WGierke FYI

The [fine-tuning notebook](https://colab.research.google.com/github/sayakpaul/ViT-jax2tf/blob/main/fine_tune.ipynb) linked in this collection closely follows the [official notebook](https://colab.research.google.com/github/google-research/vision_transformer/blob/linen/vit_jax.ipynb) for training recipes like data augmentation, LR scheduling, optimizer hyperparameters, and so on.

### CODE REVIEW

File: assets/docs/sayakpaul/collections/vision_transformer/1.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+# Collection sayakpaul/vit/1
+
+Collection of new Vision Transformer models fine-tuned on ImageNet-1k.
+
+<!-- dataset: imagenet-ilsvrc-2012-cls -->
+<!-- task: image-classification -->
+
+## Overview
+
+This collection contains seven different Vision Transformer [1] models that were fine-tuned
```
Comment (WGierke) [inline assets/docs/sayakpaul/collections/vision_transformer/1.md:10]: eight? :)

**Review by WGierke — CHANGES_REQUESTED** [review]

File: assets/docs/sayakpaul/collections/vision_transformer/1.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
+# Collection sayakpaul/vit/1
```
[the remark this answers is not in the document]
Comment (sayakpaul) [reply assets/docs/sayakpaul/collections/vision_transformer/1.md]: Done

File: assets/docs/sayakpaul/collections/vision_transformer/1.md (hunk 1 above)
Comment (sayakpaul) [reply assets/docs/sayakpaul/collections/vision_transformer/1.md]: Done

**Review by WGierke — APPROVED** [review]
Thanks a lot @sayakpaul ! The models look really promising :)

**Comment by sayakpaul** [comment]
Of course, my pleasure this is. 

I am excited because Vision Transformers will finally be on TensorFlow Hub. The community will also have the chance to choose from a large pool of Vision Transformer models, run the conversion, and use them inside TensorFlow Hub normally. 

Next up on my list is MLP-Mixer: https://arxiv.org/abs/2105.01601.

**Pull request merged**